lsif/src/correlator.ts

```diff
@@ -120,6 +124,12 @@ export class Correlator {
      */
     public exportedMonikers = new Set<MonikerId>()
 
+    private logger: Logger
+
+    constructor({ logger = createSilentLogger() }: TracingContext = { logger: createSilentLogger() }) {
```

Comment:
I think this should be enough
```suggestion
    constructor({ logger = createSilentLogger() }: TracingContext = {}) {
```